Formula/whistle.rb

```diff
@@ -3,21 +3,18 @@
 class Whistle < Formula
   desc "HTTP, HTTP2, HTTPS, Websocket debugging proxy"
   homepage "https://github.com/avwo/whistle"
-  url "https://registry.npmjs.org/whistle/-/whistle-2.9.22.tgz"
-  sha256 "bcaa19844acb08527091d7d6193bd027e45782011cf17cb13b564a5b258c392e"
+  url "https://github.com/avwo/whistle/archive/refs/tags/v2.9.24.tar.gz"
+  sha256 "e1d5892b300227c658ec7975c139cabf13c4c1b07abd9501a7ebedab52632790"
```

Comment:
Why not npm?

Reply:
The new version of npm package has not been released yet, first test it with git tag, and change it back to npm for the next version.

Reply:
We don't ship unreleases software in Homebrew.

Reply:
OK, can I roll back to the previous version?